renderer/components/TimetrackerWebsiteConncetion.tsx

```diff
@@ -174,6 +175,7 @@ const TimetrackerWebsiteConnection = () => {
       setHolidays(loggedUser?.holidays);
       setVacationsSickDays(loggedUser?.vacationsSickdays);
       setProjects(loggedUser?.yearProjects);
+      localStorage.setItem("year-projects", loggedUser?.yearProjects.join(","));
```

Comment:
You don't need to set in locastorage projects every time when components rerenders, localStorage save between reload etc.